app/src/main/java/com/fpvout/digiview/MainActivity.java

```diff
@@ -35,6 +37,11 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+        // TODO Change this to not always start the tutorial. This is only for testing purposes
```

Comment:
Should this be removed, since it's ready for merge?

Reply:
That's right, that should be removed cause I don't know how the maintainers want this to be implemented.

Reply:
Fixed in bc061bfc34aa7e67e76be7a316b4b02832d70bfd